config/olm-catalog/ibm-spectrum-scale-csi-operator/2.5.0/ibm-spectrum-scale-csi-operator.v2.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: >-
+        [{"status": {}, "kind": "CSIScaleOperator", "spec": {"provisionerNodeSelector":[{"key": "scale", "value": "true"}], "clusters":[{"secrets": "secret1", "restApi": [{"guiHost": "< Primary cluster GUI IP/Hostname>"}], "secureSslMode": false, "primary": {"primaryFs": "< Primary Filesystem >"}, "id": "< Primary Cluster ID - WARNING - THIS IS A STRING NEEDS YAML QUOTES! >"}], "pluginNodeSelector": [{"key":"scale", "value": "true"}], "resizerNodeSelector": [{"key":"scale", "value": "true"}], "attacherNodeSelector": [{"key": "scale", "value":"true"}], "snapshotterNodeSelector": [{"key": "scale", "value":"true"}]}, "apiVersion": "csi.ibm.com/v1","metadata": {"labels": {"release": "ibm-spectrum-scale-csi-operator", "app.kubernetes.io/name":"ibm-spectrum-scale-csi-operator", "app.kubernetes.io/instance": "ibm-spectrum-scale-csi-operator","app.kubernetes.io/managed-by": "ibm-spectrum-scale-csi-operator"}, "name":"ibm-spectrum-scale-csi"}}]
+    capabilities: Basic Install
+    categories: Storage
+    certified: 'false'
+    containerImage: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v2.5.0
+    createdAt: Mon Nov 01 13:55:05 IST 2021
```

Comment:
We should update createdAt to current date

Reply:
Done